integration/unplugin/README.md

```diff
@@ -84,8 +84,12 @@ interface PluginOptions {
 }
 ```
 
-- `dts`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false`
+- `emitDeclaration`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false`
 - `typecheck`: Whether to run type checking on the generated code. Default: `false`.
 - `outputExtension`: Output filename extension to use. Default: `.civet.tsx`, or uses `.civet.jsx` if `js` is `true`.
-- `js`: Whether to transpile to JS or TS. Default: `false`.
+- `ts`: Mode of transpilation of TS->JS source code. Default: `civet`. Options:
+  - `civet`: Use Civet's JS mode.
+  - `esbuild`: Use esbuild's transpiler. (Requires installing `esbuild`)
+  - `tsc`: Use the typescript compiler. (Requires installing `typescript`)
+  - `preserve`: Don't transpile TS code.
```

Comment:
Maybe mention some of limitations, like builders that don't support TS output, like Rollup. (But of course this mode will still work if the user doesn't use any TS features.)